dev-docs/plugins/bc/bc-prebid-plugin-building-adapter.md

```diff
@@ -0,0 +1,205 @@
+
+At certain points during the running of the plugin, points that correspond to the known API entry points of adapters, the plugin will query the adapter for status using one of the API entry points supported by the adapter. Depending on the outcome of the query, the Prebid plugin may or may not change its default behavior.
+
+## How to Define an Adapter to the Plugin
+
+Adapters are defined in the `adapters` prebid configuration option.  This is an optional configuration setting.  If you do want to define one or more adapters, you need to set the value of the `adapters` option to an array of one or more adapter definitions, as shown below.
+
+### Adapter Definition
+
+Each adapter definition is an object, which should contain the following fields:
+
+- `id` - this is the unique identifier of the adapter.
+    - This value should be a string; it should be the same value that the adapter is going to use to store the references to itself after it has been loaded.
+    - This is also the same identifier that the Prebid plugin is going to use to communicate with the adapter once it has been loaded.
+- `url` - this is the URL that the plugin will use to load in the adapter script.
+
+### Example Adapter Configuration
+
+```
+ options.adapters = [{id : 'myPluginAdapter', url : 'https://my-path/my-plugin-adapter.js'}];
+```
+
+### More Information
+
+For more information about adapter configuration, see  **[Prebid Options]({{site.baseurl}}/dev-docs/plugins/bc/bc-prebid-plugin-prebid-options.html#pb-adapters)**.
+
+## Adapter API
+
+By default, the Brightcove Prebid Plugin supports the following adapter API method:
+
+- `enablePrebidPlugin`
+
+If you are customizing the general behavior of the plugin and want to support other adapter entry points, you can do so in your own build of the Prebid plugin.
+
+### `enablePrebidPlugin`
+
+#### Description
+
+The plugin will call `enablePrebidPlugin` to determine if the adapter wants to block the prebid process by the plugin.  If the adapter returns a `true` status, then the Prebid plugin will continue with its normal processing of the Prebid configuration.  If, however, the adapter returns a `false` status, then the Prebid plugin will **not** continue with the prebid process and no ad will be played.
+
+#### Entry Point Prototype
+
+`enablePrebidPlugin()`
+
+This is a synchronous call.  This entry should return a value as described in the **Return Value** section below.
```

Comment:
Let's change "This entry" to "This entry point"

Reply:
fixed